Aquality.Selenium.Core/src/Aquality.Selenium.Core/Utilities/IActionRetrier.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Aquality.Selenium.Core.Utilities
+{
+    /// <summary>
+    /// Retries an action or function when handledExceptions occurs.
+    /// </summary>
+    public interface IActionRetrier
+    {
+        /// <summary>
+        /// Retries the action when one of the handledExceptions occures.
+        /// </summary>
+        /// <param name="action">Action to be applied.</param>
+        /// <param name="handledExceptions">Exceptions to be handled.</param>
+        void DoWithRetry(Action action, IEnumerable<Type> handledExceptions);
+
+        /// <summary>
+        /// Retries the action when one of the handledExceptions occures.
+        /// </summary>
+        /// <typeparam name="T">Return type of function.</typeparam>
+        /// <param name="function">Function to be applied.</param>
+        /// <param name="handledExceptions">Exceptions to be handled.</param>
+        /// <returns>Result of the function.</returns>
+        T DoWithRetry<T>(Func<T> function, IEnumerable<Type> handledExceptions);
```

Comment:
let's add an empty list of exceptions or null as the default value, I think it could be useful

Reply:
I can set default value to null and resolve it this way
var exceptionsToHandle = handledExceptions ?? new List<Type>();

Reply:
agree, let's set IEnumerable<Type> handledExceptions = null. Then I believe we could remove these definitions from IElementActionRetrier interface and ElementActionRetrier implementation

Reply:
@mialeska , already done

Reply:
done